articles/aks/load-balancer-standard.md

```diff
@@ -81,7 +81,7 @@ kubectl get service public-svc
 
 ```console
 NAMESPACE     NAME          TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)         AGE
-default       public-svc    LoadBalancer   10.0.39.110    52.156.88.187   80:32068/TCP    52s
+default       public-svc    LoadBalancer   xx.x.xx.xxx    xx.xxx.xx.xxx   80:32068/TCP    52s
```

Comment:
The CLUSTER-IP address appears to be in the [approved range](https://review.learn.microsoft.com/en-us/help/contribute/contribute-legal-guidelines?branch=main#ip-addresses) for fictitious content. It might not need to be changed.

Reply:
```suggestion
default       public-svc    LoadBalancer   10.0.39.110    203.0.113.187   80:32068/TCP    52s
```

Reply:
We don't want IPs replaced with xxx's.  The cluster IP is in the private range and is approved.  The public IP is in the approved range for documentation.  
https://review.learn.microsoft.com/en-us/help/contribute/contribute-legal-guidelines?branch=main#ip-addresses